src/index.js

```diff
@@ -0,0 +1,117 @@
+import './style.css';
+
+let tasks = [];
```

Comment:
- [ ] Following the project requirements, your project should have an array of some simple to-do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number]. You can create more properties or name these properties differently. Checkout the project requirement bellow 👇 :
<img width="693" alt="image" src="https://user-images.githubusercontent.com/77190203/155723997-d526293f-e34d-45f9-a734-b941ca827130.png">

Reply:
All good now. Well done!